classes/Project/ProjectFactory.js

```diff
@@ -703,42 +710,65 @@ export default class ProjectFactory {
    * - A dynamically fetched list of manifest items, including canvases and their annotations.
    * - All elements are embedded in the manifest object.
    */
-  static async exportManifest(projectId) {
+  static async exportManifest(projectId, creator) {
     if (!projectId) {
       throw { status: 400, message: "No project ID provided" }
     }
 
     const project = await ProjectFactory.loadAsUser(projectId, null)
+    const manifestJson = await this.fetchJson(project.manifest[0])
 
     const manifest = {
       "@context": "http://iiif.io/api/presentation/3/context.json",
       "id": `${process.env.TPENSTATIC}/${projectId}/manifest.json`,
       type: "Manifest",
       label: { none: [project.label] },
       metadata: project.metadata,
-      items: await this.getManifestItems(project),
+      items: await this.getManifestItems(project, manifestJson, creator),
     }
     return manifest
   }
 
-  static async getManifestItems(project) {
+  static async getManifestItems(project, manifestJson, creator) {
```

Comment:
`getManifestItems()` doesn't sound like it is about to save a _lot_ of items to RERUM.